portal-wire-protocol.md

```diff
@@ -124,11 +129,19 @@ selector     = 0x03
 nodes        = Container(total: uint8, enrs: List[ByteList[2048], max_length=32])
 ```
 
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflict leftovers

Reply:
thank you :heart: